doc/api/stream.md

```diff
@@ -1103,6 +1103,16 @@ added: v12.3.0
 
 Getter for the property `objectMode` of a given `Readable` stream.
 
+##### readable.readCount
+<!-- YAML
+added: CHANGEME
+-->
+
+* {number}
+
+Number of objets read in `objectMode`, otherwise number of bytes
```

Comment:
Typo: objects

But also, I think it might make more sense to flip this and mention that it’s the number of bytes read first, because object mode is the exception rather than the rule